### DISCUSSION TITLE
Is there anything on the map that indicates if a city is free or not?

### DISCUSSION THREAD

**Discussion by Doctor-Butts** [Vdauphin/HeartsAndMinds#1481] [Question and answer]
I recall in older versions, cities would have a red circle around them. Now, nothing seems to be showing on map besides intel markers.

Was this removed? Am I crazy? Is something not working somewhere?

**Comment by Vdauphin** [comment]
Hello,

Nothin changes, it has always been during the final phase

Cheers
